Tutorial.ipynb

```diff
@@ -213,55 +164,101 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "lm = cp.LabelMaker(\n",
-    "    target_entity='user_id',\n",
-    "    time_index='order_time',\n",
-    "    labeling_function=bought_product,\n",
-    "    window_size='4w',\n",
-    ")"
+    "order_products = order_products.merge(products).merge(departments)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def add_time(df):\n",
+    "    df.reset_index(drop=True)\n",
+    "    df[\"order_time\"] = np.nan\n",
+    "    days_since = df.columns.tolist().index(\"days_since_prior_order\")\n",
+    "    hour_of_day = df.columns.tolist().index(\"order_hour_of_day\")\n",
+    "    order_time = df.columns.tolist().index(\"order_time\")\n",
+    "\n",
+    "    df.iloc[0, order_time] = pd.Timestamp('Jan 1, 2015') +  pd.Timedelta(df.iloc[0, hour_of_day], \"h\")\n",
+    "    for i in range(1, df.shape[0]):\n",
+    "        df.iloc[i, order_time] = df.iloc[i - 1, order_time] \\\n",
+    "            + pd.Timedelta(df.iloc[i, days_since], \"d\") \\\n",
+    "                                    + pd.Timedelta(df.iloc[i, hour_of_day], \"h\")\n",
+    "\n",
+    "    to_drop = [\"order_number\", \"order_dow\", \"order_hour_of_day\", \"days_since_prior_order\", \"eval_set\"]\n",
+    "    df.drop(to_drop, axis=1, inplace=True)\n",
+    "\n",
+    "    return df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/nate.parsons/dev/featuretools/env/lib/python3.7/site-packages/ipykernel_launcher.py:1: UserWarning: `meta` is not specified, inferred from partial data. Please provide `meta` if the result is unexpected.\n",
```

Comment:
Is there something we can do to resolve this warning?

Reply:
I'll have look into the exact cause of that a little more, but I think it might be something we have fix in Featuretools.

Reply:
Disregard my previous comment - I didn't look closely enough and thought that warning was happening somewhere else in the notebook. Yes, I think we can fix this warning.

Reply:
@rwedge Added `meta` to the `apply` to fix the previous warning. Notebook should run without any warnings now.

Reply:
@rwedge I think everything has been resolved now - are we ready to merge?